keyboards/dz60/keymaps/jacko_keys/rules.mk

```
@@ -0,0 +1,11 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite
```

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```